### ISSUE TITLE
sys.types only allo

### ISSUE THREAD

**Issue by JayakumarEMIS** [bill-ramos-rmoswi/babelfish_postgresql_towel#17]
Hi @bill-ramos-rmoswi, there is one minor issue in sys.types , in this view working as lower case when I want to check the type is exists, I believe sys.databases and sys.tables this views working case in-sensitive, so sys.types should work case-insensitive like others, 

I have added the sample query:

`IF NOT EXISTS (SELECT * FROM sys.types where name = 'ClaimPayloadType') 
BEGIN
CREATE TYPE dbo.ClaimPayloadType AS TABLE
(
    Requestxmls NVARCHAR(max),
    Id INT
)
END
GO` 
I have created type only if not exists but issue query name in the where query is not working 

this query only working when give name value as lower case letter
SELECT * FROM sys.types where name = 'claimpayloadtype'

**Comment by staticlibs** [comment]
@JayakumarEMIS 

This problem is [fixed in the latest Babelfish](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2111), I will include the fix (for `sys.types` and `sys.views` only) into the WiltonDB update, filed [#62](https://github.com/wiltondb/wiltondb/issues/62) to track this.

**Comment by JayakumarEMIS** [comment]
Thanks @staticlibs, once fix done in wilton db update kindly let me know

**Comment by JayakumarEMIS** [comment]
Hi @staticlibs, there was one latest prerelease build available in wilton db is fix available in that version?

**Comment by staticlibs** [comment]
@JayakumarEMIS 

Yes, the fix to this is included with the [latest build](https://github.com/wiltondb/wiltondb/releases/tag/3.3-11-16-0-rc6).
